src/Features/CSharp/Portable/QuickInfo/CSharpSemanticQuickInfoProvider.cs

```diff
@@ -63,7 +66,7 @@ protected override bool ShouldCheckPreviousToken(SyntaxToken token)
         protected override ImmutableArray<TaggedText> TryGetNullabilityAnalysis(Workspace workspace, SemanticModel semanticModel, SyntaxToken token, CancellationToken cancellationToken)
         {
             // Anything less than C# 8 we just won't show anything, even if the compiler could theoretically give analysis
-            var parseOptions = (CSharpParseOptions)token.SyntaxTree.Options;
+            var parseOptions = (CSharpParseOptions)token.SyntaxTree!.Options;
```

Comment:
Quick Info is just always operating on full files, so these sorts of things are fine.